### PR TITLE
Fix 32bit-x86 typo in testsuite

### PR DESCRIPTION
**Pull request by dirkmueller**
The GOARCH here is 386 not i386. This caused a slightly odd test suite failure on that architecture:

--- FAIL: TestPlatformPrepareCommand (0.00s)
     plugin_test.go:45: Expected arg="os-arch", got "linux-s390x"
     plugin_test.go:64: Expected arg="os-arch", got "linux-s390x"

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [x] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Review by mattfarina — APPROVED** [review]
oops.

This requires one more Helm maintainer to approve before merging.

The definitive list of arch can be found in the Go source [here](https://github.com/golang/go/blob/master/src/go/build/syslist.go).

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**
